general/clients/skins.md

```diff
@@ -0,0 +1,39 @@
+---
+uid: css-custom
+title: CSS Customization
+---
+
+# CSS Customization
+
+To provide customization within Jellyfin such as changing colors or modifying behavior. Here are a collection of community scripts available. While the css scripts should work between all clients, there may be differences in behavior between the different clients. To implement these changes, go to Dashboard > General > Custom CSS.
+
+![image](https://user-images.githubusercontent.com/20715731/73392971-d1cc7d80-42a8-11ea-8552-3d311655ea37.png)
+
+## Color Modifications
+
+### Colors Codes:<br>
+Yellow: `#d0d000`<br>
+Green: `#5dd000`<br>
+Black: `#000000`<br>
+Blue: `#0000d0`<br>
+Red: `#d00000`<br>
```

Comment:
maybe just link to a hex color picker instead

Reply:
gave just the primary colors and a link to a hex chart.